src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,77 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import racinggame.dto.CarDto;
+import racinggame.dto.SnapShotDto;
+import racinggame.random.Radom;
+
+public class Cars {
+
+    private static final int DEFAULT_POSITION = 1;
+    private final SnapShotStore snapShotStore = new SnapShotStore();
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if(cars.isEmpty())throw new IllegalArgumentException("자동차가 비여 있을 수 없습니다.");
+        this.cars = cars;
+    }
+
+    public static Cars of(int count) {
+        return new Cars(carInitialization(count));
+    }
+
+    private static ArrayList<Car> carInitialization(int carCount) {
+        ArrayList<Car> cars = new ArrayList<>();
+        for (int i = 0; i < carCount; i++) {
+            cars.add(new Car(DEFAULT_POSITION));
```

Comment:
```suggestion
            cars.add(new Car());
```
`DEFAULT_POSITION`은 Car가 가지고 있는 것이 더 자연스러워 보여요
Cars가 가져간 이유가 있을까요?